python/pacsign/pacsign/pacsign.py

```diff
@@ -200,8 +206,8 @@ def main():
             False, "Invalid key manager module %s" % args.HSM_manager
         )
 
-    if args.cert_type == "RK_384":
-        common_util.assert_in_error(False, "384-bit keys not supported")
+    #if args.cert_type == "RK_384":
+    #    common_util.assert_in_error(False, "384-bit keys not supported")
```

Comment:
I would just go ahead and delete these two lines.

Reply:
Done.